fee_assessment_method.go

```diff
@@ -0,0 +1,24 @@
+package hedera
+
+type FeeAssessmentMethod bool
+
+const (
+	Inclusive FeeAssessmentMethod = false
```

Comment:
Since this is Go, you'll need to preface the name of the variable with `FeeAssessmentMethod` much like we do with `StatusOk`